bowling/player.cpp

```diff
@@ -0,0 +1,78 @@
+#include "player.hpp"
+
+#include <iomanip>
+#include <iostream>
+#include <iterator>
+#include <sstream>
+
+Player::Player() {}
+
+Player::Player(std::string name, std::vector<std::pair<int, int>> points)
+    : name_(name),
+      points_(points) {}
+
+std::string Player::getName() const {
+    return name_;
+}
+
+int Player::getScore() const {
+    return score_;
+}
+
+void Player::setScore(int score) {
+    score_ = score;
+}

```

Comment:
this is bad. Something external can set out scores to any value that is completely independent from the internal vector of points.

Reply:
moved to private block, we are using it only in Player class